### ISSUE TITLE
Button: Color contrast doesn't meet the WCAG AA contrast ratio

### ISSUE THREAD

**Issue by c0lt** [primefaces/primeuix#136]
In the `Button` component (and maybe others that are using the predefined `severity` prop), the color contrast between text and its background doesn't meet the WCAG AA contrast ratio.

<img width="1216" height="486" alt="Image" src="https://github.com/user-attachments/assets/48b7b583-e99d-4242-99c5-ad7d2689ec6c" />

The problem exists also for `raised`, `outlined` or `text` variants.

**Comment by cagataycivici** [comment]
I think it is better to customize it on your end, Prime has numerous themes. This is a huge change for us since Figma needs to be updated as well and backward compatibility issues will occur from the community so better to customize on your end.
